shared_model/validators/transaction_validator.hpp

```diff
@@ -36,11 +36,14 @@ namespace shared_model {
     /**
      * Visitor used by transaction validator to validate each command
      * @tparam FieldValidator - field validator type
+     * @note this class is not a thread safe and never going to
+     * so copy constructor is disabled explicitly
      */
     template <typename FieldValidator>
     class CommandValidatorVisitor
         : public boost::static_visitor<ReasonsGroupType> {
      public:
+      CommandValidatorVisitor(const CommandValidatorVisitor &) = delete;
```

Comment:
Please, also disable a copy assignment operator, as it is generated, even if copy ctor is disabled